assets/pages/monthly-contributions/reducers/reducers.js

```diff
@@ -44,6 +45,11 @@ export type CombinedState = {
   csrf: CsrfState,
 };
 
+export type PageState = {
+  common: CommonState,
+  page: CombinedState,
+}
+
 // ----- Reducers ----- //
 
 function createMonthlyContribReducer(amount: number, currency: Currency, country: IsoCountry) {
```

Comment:
I don't think `currency` and `country` need to be handled by this reducer any more, as they exist in the `common` state.